app/_data/versions.yml

```diff
@@ -1,34 +1,61 @@
-- edition: "kuma"
-  version: "1.1.6"
-  release: "1.1.x"
-- edition: "kuma"
-  version: "1.2.3"
-  release: "1.2.x"
-- edition: "kuma"
-  version: "1.3.1"
-  release: "1.3.x"
-- edition: "kuma"
-  version: "1.4.1"
-  release: "1.4.x"
-- edition: "kuma"
-  version: "1.5.5"
-  release: "1.5.x"
-- edition: "kuma"
-  version: "1.6.5"
-  release: "1.6.x"
-- edition: "kuma"
-  version: "1.7.5"
-  release: "1.7.x"
-- edition: "kuma"
-  version: "1.8.4"
-  release: "1.8.x"
-- edition: "kuma"
-  version: "2.0.3"
-  release: "2.0.x"
-- edition: "kuma"
-  version: "2.1.1"
-  release: "2.1.x"
+- edition: kuma
+  version: 1.2.3
+  release: 1.2.x
+  releaseDate: "2021-06-17"
+  endOfLifeDate: "2022-06-17"
+  branch: release-1.2
+- edition: kuma
+  version: 1.3.1
+  release: 1.3.x
+  releaseDate: "2021-08-26"
+  endOfLifeDate: "2022-08-26"
+  branch: release-1.3
+- edition: kuma
+  version: 1.4.1
+  release: 1.4.x
+  releaseDate: "2021-11-22"
+  endOfLifeDate: "2022-11-22"
+  branch: release-1.4
+- edition: kuma
+  version: 1.5.4
+  release: 1.5.x
+  releaseDate: "2022-02-24"
+  endOfLifeDate: "2023-02-24"
+  branch: release-1.5
```

Comment:
should we be including EOL versions?

Reply:
Yes because we still keep docs around for EOL versions.

Reply:
Not real sure what's the reason for this but it's not something I want to fix now :)